doc/api/vm.md

```diff
@@ -38,6 +38,10 @@ The options when creating a script are:
   script's methods.
 - `timeout`: a number of milliseconds to execute `code` before terminating
   execution. If execution is terminated, an [`Error`][] will be thrown.
+- `breakEvalOnSigint`: if `true`, the execution will be terminated when
+  `SIGINT` (Ctrl+C) is received. This will disable existing handlers for the
+  event that have been attached via `process.on("SIGINT")` permanently.
```

Comment:
@Fishrock123 Changed this so that existing `SIGINT` handlers are re-attached after running a script with `breakOnSigint` by adding bdf0661721d8e7911b7d98db2e9b967c41a59055 to this PR.